assets/scss/base/_lists.scss

```diff
@@ -14,6 +25,38 @@ ul, nav ul {
 	}
 }
 
+/*
+Numbered List
+
+Defining sections in markup with numbers associated against them using `.list--numbered` on the container and `.numbered` on the element to identify its position
+
+Markup:
+<fieldset class="list--numbered">
+  <legend>Section 1</legend>
+  <div class="form-field-group numbered">
+    <label class="bold-small" for="example-1">
+      Question 1
+      <div class="form-field">
+        <input type="text" class="form-control" id="example-1" />
+      </div>
+    </label>
+  </div>
+</fieldset>
+<fieldset class="list--numbered">
+  <legend>Section 2</legend>
+  <div class="form-field-group numbered">
+    <label class="bold-small" for="example-2">
+      Question 1
+      <div class="form-field">
+        <input type="text" class="form-control" id="example-2" />
+      </div>
+    </label>
+  </div>
+</fieldset>
+
+Styleguide Lists.Numbered List
+*/
+
 .list--numbered {
```

Comment:
I think we need to raise an issue around this - `list--numbered` is poorly named if it is not used with a `list`. It feels like the `BEM` convention is slightly confused